src/Fixer/Basic/BracesFixer.php

```diff
@@ -429,7 +439,11 @@ function ($item) {
                     || !($nextToken->isWhitespace() && $nextToken->isWhitespace(" \t"))
```

Comment:
last changes undid the request;
I like when a condition in the form;
```php

if ($a || $b && $c) {
    echo 1;
}
```

is written out explicit as;
```php

if ($a || ($b && $c)) {
    echo 1;
}
```

maybe one day I make a fixer for it ;)